qparam.go

```diff
@@ -195,6 +196,7 @@ func (r *Reader) readSlice(values []string, slice reflect.Value) error {
 // which altogether caused the actual failure.
 type MultiError interface {
 	error
+	fmt.Formatter
```

Comment:
It is not necessary to extend the `MultiError` interface with `Formatter`. Just implementing `Format` for `multiError` should be enough.

Reply:
It's true that interfaces are implicitly satisfied but making this explicit does two things:
1. documents the intent to a reader
2. enforces it (if for instance fmt.Formatter were to have a new function added)

As such, I don't see the harm in listing it.

Reply:
OK